CMakeLists.txt

```diff
@@ -793,7 +793,18 @@ if(AVIF_BUILD_APPS)
         set_target_properties(avifdec PROPERTIES LINKER_LANGUAGE "CXX")
     endif()
     target_link_libraries(avifdec avif_apps)
-
+    if(WIN32)
+        foreach(PROG IN LISTS avifenc avifdec)
+            add_custom_command(
+                TARGET ${PROG}
+                POST_BUILD
+                COMMAND "mt.exe" -manifest \"${CMAKE_SOURCE_DIR}\\src\\utf8.manifest\"
+                        -inputresource:\"$<TARGET_FILE:${PROG}>\"\;\#1 -outputresource:\"$<TARGET_FILE:${PROG}>\"\;\#1
+                COMMENT "Adding manifest..."
```

Comment:
When I tested this, I didn't need to use add_custom_command(). It sufficed to just add utf8.manifest as a source file.

I will try to find my changes.